bundles/org.openhab.binding.nanoleaf/README.md

```diff
@@ -127,13 +136,15 @@ The controller bridge has the following channels:
 | rhythmState         | Switch    | Connection state of the rhythm module                                  | Yes       |
 | rhythmActive        | Switch    | Activity state of the rhythm module                                    | Yes       |
 | rhythmMode          | Number    | Sound source for the rhythm module. 0=Microphone, 1=Aux cable          | No        |
+| swipe               | Trigger   | [Canvas / Shapes Only] Detects Swipes over the panel. LEFT, RIGHT, UP, DOWN events are supported.          | YES        |
+
 
 A lightpanel thing has the following channels:
 
 | Channel             | Type      | Description                                                            | Read Only |
 |---------------------|-----------|------------------------------------------------------------------------|-----------|
 | color               | Color     | Color of the individual light panel                                    | No        |
-| tap                 | Trigger   | [Canvas Only] Sends events of gestures. Currently, these are SHORT_PRESSED and DOUBLE_PRESSED events.                       | Yes       |
+| tap                 | Trigger   | [Canvas / Shapes Only] Sends events of gestures. SHORT_PRESSED and DOUBLE_PRESSED events are supported.  | Yes       |
```

Comment:
Done, even though before doing so it would render fine :-)
![image](https://user-images.githubusercontent.com/5937600/134292933-b7741cd0-9b76-48f9-9eab-befbd21fd2fc.png)